src/checks/#cc4a#modern_language.clas.xml

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_CLAS" serializer_version="v1.0.0">
+ <asx:abap version="1.0" xmlns:asx="http://www.sap.com/abapxml"/>
+</abapGit>
```

Comment:
All metadata seems to be missing here (compare this to any other *.clas.xml file), and so I cannot import this into any other system to test it (error from Steampunk abapGit: "Invalid parameter OBJECT/OBJECTCLASS"). Maybe try to export this again?

Reply:
I have added the metadata, hope it does work now